charts/cert-manager-1.14.5/README.md

```diff
@@ -25,11 +25,7 @@ $ kubectl apply -f https://github.com/cert-manager/cert-manager/releases/downloa
 To install the chart with the release name `my-release`:
 
 ```console
-## Add the Jetstack Helm repository
-$ helm repo add jetstack https://charts.jetstack.io
-
-## Install the cert-manager helm chart
-$ helm install my-release --namespace cert-manager --version v1.14.5-4-ge5f0f61d7f1da9 jetstack/cert-manager
+ocne application install --name cert-manager --release cert-manager --namespace cert-manager
```

Comment:
`--version 1.14.5` needs to be specified. Otherwise, latest version of cert-manager in the catalog would be installed instead.